key_management/src/keystore.rs

```diff
@@ -557,4 +559,27 @@ mod test {
         let ks = KeyStore::new(KeyStoreConfig::Persistent(keystore_location)).unwrap();
         ks.flush().unwrap();
     }
+
+    impl quickcheck::Arbitrary for KeyInfoJson {
```

Comment:
We want to generate `KeyInfo`s. The `KeyInfoJson` structure is just there to help with the JSON encoding.